app/views/fields/action_text/_index.html.erb

```diff
@@ -0,0 +1 @@
+<%= field.to_s %>
```

Comment:
Maybe this should not display the all the content on the index view by default? Or have some option around this